pkg/executor/executor.go

```diff
@@ -181,7 +181,17 @@ func DoBuild(k KanikoBuildArgs) (v1.Image, error) {
 	return nil, err
 }
 
+type withUserAgent struct {
+	t http.RoundTripper
+}
+
+func (w *withUserAgent) RoundTrip(r *http.Request) (*http.Response, error) {
+	r.Header.Set("User-Agent", fmt.Sprintf("kaniko/version.Version()"))
```

Comment:
I think this should be `fmt.Sprintf("kaniko/%s", version.Version())`

Reply:
Yes, yes it should :)